Content.Server/GameObjects/EntitySystems/ActSystem.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Linq;
+using JetBrains.Annotations;
+using Robust.Shared.GameObjects.Systems;
+using Robust.Shared.Interfaces.GameObjects;
+using Content.Shared.GameObjects;
+
+namespace Content.Server.GameObjects.EntitySystems
+{
+    /// <summary>
+    /// This interface gives components behavior when being "triggered" by timer or other conditions
+    /// </summary>
+    public interface IDestroyAct
+    {
+        /// <summary>
+        /// Called when one object is triggering some event
+        /// </summary>
+        void Destroy(DestructionEventArgs eventArgs);
+    }
+
+    public class DestructionEventArgs : EventArgs
+    {
+        public IEntity Owner { get; set; }
+        public DamageType TypeOfDamage { get; set; }
```

Comment:
I'm kind of on the edge of whether this should even carry damage, because it's too contextual to make sense of in a "general" case like this.

Reply:
It carries damage for second-hand explosions. If we don't need it - it will go away.